docs/contributor_guide/index.rst

```diff
@@ -43,4 +43,16 @@ We have a monthly contributor meeting every second Thursday of the month at
 All are welcome! Please reach out via
 `Gitter <https://gitter.im/fairlearn/community>`_ and we will forward the
 invitation to you.
-We keep notes from each meeting in the :ref:`developer_call_notes`.
\ No newline at end of file
+We keep notes from each meeting in the :ref:`developer_call_notes`.
+
+.. _sociotechnical_deep_dive:
+
+Sociotechnical deep dives
+-------------------------
+
+We are currently experimenting with weekly sociotechnical deep dives.
+They take place on Thursday at 11am New York City time whenever there is no
```

Comment:
I know NYC thinks highly of itself, but isn't it "Easter Standard Time" ?

Reply:
That is in fact (and quite surprisingly) not true! :-) 
We are doing this at EDT starting next week, not EST. To be consistent with daylight savings this just says NYC. We really have to be clear about this, since it's already caused confusion due to Europe and the US not being able to agree when daylight savings are observed.

Reply:
Changed this to Eastern Time which should always be accurate.